src/constants.js

```diff
@@ -174,6 +174,17 @@ export const emotesCategoryIds = {
   FRECENTS: 'frecents',
 };
 
+export const defaultFrequentEmotes = {
```

Comment:
constants should never be camelCase.

for enum values, use `PascalCase`.
for everything else, use `SNAKE_CASE`.